helm/generic-node/templates/archive-node.yaml

```diff
@@ -0,0 +1,171 @@
+---
+{{if .Values.archive.enabled}}
+apiVersion: apps/v1
+{{- if .Values.features.persistence.enabled}}
+kind: StatefulSet
+{{- else}}
+kind: Deployment
+{{- end}}
+metadata:
+  name: {{.Values.name}}-archive
+  labels:
+    app: {{.Values.name}}-archive
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{.Values.name}}-archive
+  serviceName: "nginx"
+  template:
+    metadata:
+      labels:
+        app: {{.Values.name}}-archive
+        testnet: {{.Values.testnetName}}
+        role: archive-node
+        class: "archive-node"
+        version: {{.Values.archive.image.tag | quote}}
+        syncStatus: INIT
+      annotations:
+        prometheus.io/scrape: 'true'
+        prometheus.io/port: {{.Values.ports.metrics | quote}}
+        prometheus.io/path: '/metrics'
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                - key: "cloud.google.com/gke-preemptible"
+                  operator: In
+                  values: ["true"]
+      initContainers:
+      {{- if .Values.features.privateKeySecret.enabled}}
+      - name: fix-perms
+        image: busybox:latest
+        command: [ 'sh', '-c', 'for dir in keys echo-keys faucet-keys; do [ -d /$dir ] && /bin/cp /$dir/* /wallet-keys; done; /bin/chmod 0700 /wallet-keys']
+        volumeMounts:
+        - mountPath: "/keys/"
+          name: private-keys
+          readOnly: true
+        - mountPath: /wallet-keys
+          name: wallet-keys
+      {{- end }}
+      containers:
+      - name: mina-archive
+        image:  "{{.Values.archive.image.repository }}:{{.Values.archive.image.tag}}"
+        imagePullPolicy: {{.Values.archive.image.pullPolicy}}
+        {{- if .Values.features.customEntrypoint.enabled}}
+        command: [ {{ .Values.features.customEntrypoint.command}} ]
+        {{- end }}
+        volumeMounts:
+        - name: destination-libp2p
+          mountPath: /root/libp2p-keys
+          readOnly: true
+        - name: config-dir
+          mountPath: "/root/.mina-config"
+        {{- if .Values.features.uploadBlocksToGCloud.enabled}}
+        - name: gcloud-keyfile
+          mountPath: "/gcloud/"
+        {{- end }}
```

Comment:
Remove the whole volumeMounts section, the archive node does not require any volumes